### ISSUE TITLE
[question] Gateway aggregates data based on what?

### ISSUE THREAD

**Issue by aliok** [weaveworks/prom-aggregation-gateway#14]
> Prometheus Aggregation Gateway is a aggregating push gateway for Prometheus. As opposed to the official Prometheus Pushgateway, this service aggregates the sample values it receives.

This is great, but how is the data aggregated? In what dimensions, what slices, what buckets?

What is the default and is there a way to configure aggregation with custom way?

Also, any Docker images and instructions on how to use them?

**Comment by gissurWXNC** [comment]
I've been looking at this also and have been pretty dismayed by the dearth of documentation.
I did manage to dig up the docker image: https://quay.io/repository/weaveworks/prom-aggregation-gateway

What I'm missing is some way to deploy it. On a lark, I tried replacing the image tag on the `stable/prometheus-pushgateway` helm chart - but the Aliveness and Readiness probes fail and the container never comes up.
